packages/node-utils/src/http.ts

```diff
@@ -65,7 +67,7 @@ type BaseEvents = {
 export class HttpServer<T extends EventMap> extends TypedEmitter<T & BaseEvents> {
     server: http.Server;
```

Comment:
Nit: In this case `server` is public by default, if we start to declare `public` explicitly it would be nice to be consistent.